file-share-service-desktop-client/Modules/Admin/JobViewModels/SetExpiryDateJobViewModel.cs

```diff
@@ -20,7 +21,11 @@ protected internal override Task OnExecuteCommand()
         }
         protected override bool CanExecute()
         {
-            throw new System.NotImplementedException();
+            ValidationErrors.Clear();
+
+            ValidationErrors = job.ErrorMessages;
+
+            return ValidationErrors.Any();
```

Comment:
Should this be
````
return !ValidationErrors.Any();
````

Also, do you think it's worth moving `CanExecute()` to the base class. I notice the implementation of `CanExecute()` is identical in `ErrorDeserializingJobsJobViewModel`, `AppendAclJobViewModel` and `SetExpiryDateJobViewModel`

Reply:
Made the changes as suggested.